### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by stdavis**
This allows you to destroy a layer-selector and create a new one on the
same map.

### CODE REVIEW

File: LayerSelector.js [hunk 1]
```diff
@@ -725,6 +725,24 @@ define([
...
+        },
+        /**
+         * Remove layers from the map before destroying the widget.
+         */
+        destroy: function () {
+            console.log('layer-selector:destroy', arguments);
+
+            var managedLayers = this.get('managedLayers');
+            for (var layerName in managedLayers) {
+                if (managedLayers.hasOwnProperty(layerName)) {
```
Comment (steveoh) [inline LayerSelector.js]: I would maybe try to reduce the arrow functionness of this in the future and consider object.keys but it works for me.

File: resources/LayerSelector.css [hunk 2]
```diff
@@ -48,7 +48,6 @@
   -webkit-box-shadow: 0 1px 5px rgba(0,0,0,0.4);
   box-shadow: 0 1px 5px rgba(0,0,0,0.4);
   background: #fff;
-  -webkit-border-radius: 5px;
```
Comment (steveoh) [inline resources/LayerSelector.css]: yes stylus would add this but if you want to use the css without touching stylus i would think you might need this or is the vendor prefix not necessary anymore? Should we remove all of the prefixes?

Comment (stdavis) [inline resources/LayerSelector.css]: I didn't actually touch this file by hand. I just assumed that when I ran the grunt stylus plugin it didn't add that prefix. I don't believe that prefix is used anymore.

File: LayerSelector.js (hunk 1 above)
Comment (stdavis) [inline LayerSelector.js]: Thanks for reminding me about that. I switched it to `Object.keys()`.

**Pull request merged**